packages/angular/package.json

```diff
@@ -104,7 +103,7 @@
     "karma-parallel": "0.3.1",
     "lit-element": "2.3.1",
     "lit-html": "1.2.1",
-    "ng-packagr": "11.0.2",
+    "ng-packagr": "11.2.4",
```

Comment:
Bumped as this removes part of the stylus node 14 issues mentioned above https://github.com/stylus/stylus/issues/2534